pkg/okctl/okctl.go

```diff
@@ -199,6 +199,34 @@ func (o *Okctl) ClientServices(spin spinner.Spinner) (*clientCore.Services, erro
 	}, nil
 }
 
+func (o *Okctl) managedPolicyService(outputDir string, spin spinner.Spinner) client.ManagedPolicyService {
+	return clientCore.NewManagedPolicyService(spin,
+		rest.NewManagedPolicyAPI(o.restClient),
+		clientFilesystem.NewManagedPolicyStore(
+			clientFilesystem.Paths{
+				OutputFile:         config.DefaultPolicyOutputFile,
+				CloudFormationFile: config.DefaultPolicyCloudFormationTemplateFile,
+				BaseDir:            path.Join(outputDir, config.DefaultPolicyBaseDir),
+			},
+			o.FileSystem,
+		),
+		console.NewManagedPolicyReport(o.Err, spin),
+	)
+}
+
+func (o *Okctl) serviceAccountService(outputDir string, spin spinner.Spinner) client.ServiceAccountService {
```

Comment:
![image](https://user-images.githubusercontent.com/562343/110471878-f9615c80-80dc-11eb-9cf8-80975043dfd2.png)

In all fairness, nice addition.